recipes/orthodb/meta.yaml

```
@@ -0,0 +1,51 @@
+requirements:
+  build:
+    - python >=3.9
+    - bash >=4.1
+  host:
+    - python >=3.9
+    - pip
+    - bash >=4.1
+  run:
+    - python >=3.9
+    - bash >=4.1
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)